### PR TITLE
October 2020 Update

### PR DESCRIPTION
**Pull request by crpietschmann**
These are the updates for the October 2020 Update for Issue: #188

### CODE REVIEW

**Comment by sabbour** [comment]
On https://github.com/microsoft/MCW-Cloud-native-applications/blob/fd10fca9472c7e493263265b8308d754de61fbf4/Hands-on%20lab/HOL%20step-by-step%20-%20Cloud-native%20applications%20-%20Developer%20edition.md#task-4-deploy-a-service-using-a-helm-chart would it be possible to expand a little on what we're deploying with a Helm chart? Perhaps a preamble of how we're intending to package this into a chart before jumping into the commands?

**Comment by sabbour** [comment]
As a post task to this https://github.com/microsoft/MCW-Cloud-native-applications/blob/fd10fca9472c7e493263265b8308d754de61fbf4/Hands-on%20lab/HOL%20step-by-step%20-%20Cloud-native%20applications%20-%20Developer%20edition.md#task-2-increase-service-instances-beyond-available-resources can we add something around enabling the cluster autoscaler to remediate this issue?

**Comment by crpietschmann** [comment]
> As a post task to this https://github.com/microsoft/MCW-Cloud-native-applications/blob/fd10fca9472c7e493263265b8308d754de61fbf4/Hands-on%20lab/HOL%20step-by-step%20-%20Cloud-native%20applications%20-%20Developer%20edition.md#task-2-increase-service-instances-beyond-available-resources can we add something around enabling the cluster autoscaler to remediate this issue?

I agree that configuring cluster autoscaling is an exercise that would help the student better understand what AKS has to offer. However, this wasn't in the original plan for this update cycle. This is something we would need to consider for a future update.

File: Hands-on lab/HOL step-by-step - Cloud-native applications - Developer edition.md [hunk 1]
```diff
@@ -38,28 +38,31 @@ Microsoft and the trademarks listed at https://www.microsoft.com/en-us/legal/int
     - [Task 4: Create Docker images](#task-4-create-docker-images)
     - [Task 5: Run a containerized application](#task-5-run-a-containerized-application)
     - [Task 6: Setup environment variables](#task-6-setup-environment-variables)
-    - [Task 7: Run several containers with Docker compose](#task-7-run-several-containers-with-docker-compose)
-    - [Task 8: Push images to Azure Container Registry](#task-8-push-images-to-azure-container-registry)
-    - [Task 9: Setup CI Pipeline to Push Images](#task-9-setup-ci-pipeline-to-push-images)
-  - [Exercise 2: Deploy the solution to Azure Kubernetes Service](#exercise-2-deploy-the-solution-to-azure-kubernetes-service)
+    - [Task 7: Push images to Azure Container Registry](#task-7-push-images-to-azure-container-registry)
+    - [Task 8: Setup CI Pipeline to Push Images](#task-8-setup-ci-pipeline-to-push-images)
+    - [Task 2: Provision Azure Database Migration Service](#task-2-provision-azure-database-migration-service)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Cloud-native applications - Developer edition.md:43]: No line in TOC for Exercise 2 and Exercise 2, Task 1.
Exercise 1 has tasks 1-8, then 2 & 3

**Pull request merged**